src/backend/utils/inactive-blog-filter.js

```diff
@@ -92,20 +86,19 @@ function update() {
     let linesRead = 0;
 
     feedUrlList.forEach(async feedItem => {
-      const feed = await feedParser(feedItem);
+      const feed = await parser(feedItem);
       const feedUrl = feedItem.url;
 
       let recentPostDate = new Date();
 
-      // In case of invalid/ dead feeds
-      if (typeof feed[0] !== 'undefined') {
+      if (feed && typeof feed[0] !== 'undefined') {
         recentPostDate = new Date(feed[0].date);
 
         // Check if the blog is inactive
         // We convert the dateDiff(result) from ms in days
         const timeDiff = Math.ceil(dateDiff(recentPostDate) / (1000 * 3600 * 24));
 
-        if (timeDiff > process.env.BLOG_INACTIVE_TIME) {
+        if (timeDiff > (process.env.BLOG_INACTIVE_TIME || 365)) {
```

Comment:
Actually, this is the exact change that was needed to pass my CI tests! Check out #396

Reply:
Oh OK. Do you still want to at least make a comment for `365`, since I couldn't tell what the number was at first glance meaning later contributors might also have that problem in the future.

Reply:
Great idea! Adding one right in!